legal-api/migrations/versions/b14f77400786_xml_payloads.py

```diff
@@ -0,0 +1,43 @@
+"""xml_payloads
+
+Revision ID: b14f77400786
+Revises: feb206b2ce65
+Create Date: 2024-07-17 23:46:20.873233
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'b14f77400786'
+down_revision = 'feb206b2ce65'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table('xml_payloads',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('payload', sa.TEXT, nullable=False),    
+    sa.Column('created_date', sa.TIMESTAMP(timezone=True), nullable=False),
+    sa.PrimaryKeyConstraint('id')
+    )
+
+    op.create_table('furnishing_groups',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('xml_payload_id', sa.Integer(), nullable=False),
```

Comment:
Can it be nullable because we might add more links other than `xml_payload_id`? 🤔 💭

Reply:
And it's also possible that we only have the `id` value populated

Reply:
Updated